utils.go

```diff
@@ -53,6 +53,23 @@ func IsDocumentation(path string) bool {
 	return data.DocumentationMatchers.Match(path)
 }
 
+func IsImage(file string) bool {
```

Comment:
Maybe it's better call the parameter `path` instead `file` to be consistent with the rest of the functions